packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/edit/edit.html

```diff
@@ -442,9 +441,9 @@
                                         data-ng-disabled="!$ctrl.isConditionGroupValid(conditionsGroup)"
                                         data-ng-click="$ctrl.onConditionAddBtnClick(conditionsGroup)"
                                     >
-                                        <i
-                                            class="ovh-font ovh-font-ajouter"
-                                        ></i>
+                                        <span
+                                            class="oui-icon oui-icon-success group-number__dialplan--check-btn"
```

Comment:
```suggestion
                                            class="oui-icon oui-icon-success group-number__dialplan--check-btn"
                                            aria-hidden="true"
```
and add a span or a description to know what the button is meant for